.github/workflows/ci.yml

```diff
@@ -145,13 +145,13 @@ jobs:
       - name: Run Extension Unit Tests (Continue on Error) # {{SQL CARBON EDIT}} Run extension unit tests (continue on error)
         id: electron-extension-unit-tests-continue-on-error
         run: DISPLAY=:10 ./scripts/test-extensions-unit.sh
-        continueOnError: true
-        condition: and(succeeded(), eq(variables['EXTENSION_UNIT_TESTS_FAIL_ON_ERROR'], 'false'))
+        continue-on-error: true
+        if: ${{ success() && env.EXTENSION_UNIT_TESTS_FAIL_ON_ERROR == 'false' }}
```

Comment:
Ques: Why do we want to put env variable condition if we want to Continue on error? It's never going to run alongside the next task as both conditions are contradictory.

Reply:
For ADO pipelines that's the point - continueOnError doesn't allow for evaluated conditions and so in order to have some versions of the pipeline continue on error and some not then we have to make two separate tasks that have different IF conditions

See https://developercommunity.visualstudio.com/t/variablesexpressions-dont-work-with-continueonerro/1187733

But for the CI workflows you're right - we don't need these at all since we don't have multiple "CI" pipelines. So you can just revert the changes in this file. 

(and maybe Github workflows don't have that same problem regardless, I wouldn't be surprised)